src/database/queries.tsx

```diff
@@ -304,21 +305,40 @@ export const getAllUpcomingAppointmentsForClient = async (
   }
 };
 
-export const getAllCalendlyLinks = async (): Promise<CalendlyLink[]> => {
```

Comment:
[ACTION]: Need to update `getAllUpcomingAppointmentsForClient ` query. Querying for links works, but upcoming appointments are still relying on the root calendlyLinks collection instead of the link field in caseTypes